### DISCUSSION TITLE
How to constraint content size of  a `TabPanel` by size of the `TabPanel`

### DISCUSSION THREAD

**Discussion by sharpchen** [primefaces/community#1451] [PrimeVue] [answered]
**I am using tailwind primevue with preset wind.**
https://stackblitz.com/edit/vav812?file=src%2FApp.vue

I'd like to make content of a `TabPanel` forced to fit the specified size of a `TabPanel`. As far as I can tell, size of `TabPanel` dynamically fits its content size, I tried to set static size for `TabPanel` and using flexbox in css but didn't work.

**Comment by tugcekucukoglu** [comment, marked as answer]
Could you try to add:

```
<TabView
    :pt="{
      tabpanel: {
        content: 'tab',
      },
    }"
  >
```
